packages/bpk-component-breadcrumb/src/BpkBreadcrumbItem.js

```diff
@@ -0,0 +1,101 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2018 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/* @flow */
+
+import PropTypes from 'prop-types';
+import React, { type Node } from 'react';
+import BpkLink from 'bpk-component-link';
+import BpkText from 'bpk-component-text';
+import { cssModules } from 'bpk-react-utils';
+import { withRtlSupport } from 'bpk-component-icon';
+import ArrowRight from 'bpk-component-icon/lg/arrow-right';
+
+import STYLES from './BpkBreadcrumbItem.scss';
+
+const getClassName = cssModules(STYLES);
+
+export type Props = {
+  children: Node,
+  active: boolean,
+  href: ?string,
+  className: ?string,
+  'aria-current': ?string,
```

Comment:
This isn't correct. `aria-current` should be `string` since we fill it in with `defaultProps`

Reply:
With Flow, the `?` means it's optional. As a rule of thumb, if the default value for a prop is `null`, the Flow type is optional.

Reply:
As default prop is now null, `?string` is correct so this is resolved :)